File: Intro_POO/Intro_POO_python/Fruta.py
class Fruta:
    def __init__(self, nombre, tipo, vitaminas):
        self.nombre = nombre
        self.tipo = tipo
        self.vitaminas = vitaminas
        self.nroVitaminas = len(vitaminas)

    def getNombre(self):
        return self.nombre

    def getTipo(self):
        return self.tipo

    def getNroVitaminas(self):
        return self.nroVitaminas

    def getVitaminas(self):
        return self.vitaminas

    def __str__(self):
        return f"Fruta: {self.nombre} | Tipo: {self.tipo} | nroVitaminas: {self.nroVitaminas} | Vitaminas: {self.vitaminas}"


if __name__ == "__main__":
    f1 = Fruta("Naranja", "Cítrica", ["A", "C"])
    f2 = Fruta("Manzana", "Dulce", ["B1", "B2", "C"])

    print(f1)
    print(f2)

File: Intro_POO/Ejercicio_13/Fruta.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.introduccionpoo;

/**
 *
 * @author HP-RYZEN 5
 */
//Nicol Yanina Ticona Aduviri
//R.U: 1855987
import java.util.Arrays;

class Fruta {

    private String nombre;
    private String tipo;
    private int nroVitaminas;
    private String[] vitaminas;

    public Fruta(String nombre, String tipo, String[] vitaminas) {
        this.nombre = nombre;
        this.tipo = tipo;
        this.nroVitaminas = vitaminas.length;
        this.vitaminas = vitaminas;
    }

    public String getNombre() {
        return nombre;
    }

    public String getTipo() {
        return tipo;
    }

    public int getNroVitaminas() {
        return nroVitaminas;
    }

    public String[] getVitaminas() {
        return vitaminas;
    }

    // para mostrar info
    @Override
    public String toString() {
        return "Fruta: " + nombre + " | Tipo: " + tipo
                + " | nroVitaminas: " + nroVitaminas
                + " | Vitaminas: " + Arrays.toString(vitaminas);
    }
}
